src/credentials/BUILD.gn

```diff
@@ -32,6 +32,8 @@ static_library("credentials") {
     "DeviceAttestationVerifier.cpp",
     "DeviceAttestationVerifier.h",
     "GenerateChipX509Cert.cpp",
+    "GroupDataProvider.h",
+    "StaticGroupDataProvider.cpp",
```

Comment:
StatucGroupDataProvider should probably be under `examples`

Reply:
StaticGroupDataProvider.cpp moved to credentials/examples/GroupDataProviderExample.cpp